packages/datadog-plugin-cucumber/src/index.js

```diff
@@ -213,10 +284,11 @@ class CucumberPlugin extends CiPlugin {
   }
 
   startTestSpan (testName, testSuite, extraTags) {
+    const testSuiteSpan = this.testSuiteSpanByPath[testSuite]
     return super.startTestSpan(
       testName,
       testSuite,
-      this.testSuiteSpan,
```

Comment:
ℹ️ now there can be multiple test suite spans active at the same time, that's why we can use `this.testSuiteSpan`